src/foundation.cs

```diff
@@ -7179,6 +7179,12 @@ interface NSString2 : NSSecureCoding, NSMutableCopying
 		[Static]
 		[Export ("localizedUserNotificationStringForKey:arguments:")]
 		string GetLocalizedUserNotificationString (string key, [Params] [NullAllowed] NSObject [] arguments);
+
+		[Export ("getParagraphStart:end:contentsEnd:forRange:")]
+		void GetParagraphPositions (out nuint paragraphStartPosition, out nuint paragraphEndPosition, out nuint contentsEndPosition, NSRange range);
```

Comment:
@chamons it might be a nice addition but it breaks subclassing, so this one should be exposed anyway (at least as `protected`).